### PR TITLE
Cult bolas no longer affect holy-proof individuals.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Cult bola's will now pass through anyone with holy magic protection.

## Why It's Good For The Game

Cult bolas are an incredibly powerful tool, and despite being magic are not affected by anti-holy magic.
In practice this means that holding a bible in one of your hands will prevent cult bolas from protecting you.

## Changelog
:cl:
balance: Cult bolas no longer affect people with holy magic protection.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/cult/cult_items.dm [hunk 1]
```diff
@@ -261,6 +261,12 @@
 /obj/item/restraints/legcuffs/bola/cult/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
 	if(iscultist(hit_atom))
 		return
+	if(ismob(hit_atom))
+		var/mob/M = hit_atom
+		var/anti_magic_source = M.anti_magic_check(holy = TRUE)
+		if(anti_magic_source)
```
Comment (Archanial) [inline code/modules/antagonists/cult/cult_items.dm]:
```suggestion
		if(M.anti_magic_check(holy = TRUE))
```

**Comment by Isy232** [comment]
I agree cult bolas should be nerfed a little, but #6126 does this better in my opinion.

**Comment by PowerfulBacon** [comment]
> I agree cult bolas should be nerfed a little, but #6126 does this better in my opinion.

This brings them in line with all other cultist equipment that has magic effects, stuff that can't affect cultists generally can't affect holy individuals too

**Comment by Isy232** [comment]
> > I agree cult bolas should be nerfed a little, but #6126 does this better in my opinion.
> 
> This brings them in line with all other cultist equipment that has magic effects, stuff that can't affect cultists generally can't affect holy individuals too

I guess your right, both of these PR's could be merged together.

I orgininally didnt think that this would be enough of a nerf to cult bolas as only 1-2 people could be immune to them at most, but this seems fine to merge with the other nerf PR.

**Review by Archanial — APPROVED** [review]

**Pull request merged**